crates/bevy_render/src/render_resource/uniform_buffer.rs

```diff
@@ -139,6 +141,17 @@ impl<T: ShaderType + WriteInto> UniformBuffer<T> {
     }
 }
 
+impl<'a, T: ShaderType + WriteInto> IntoBinding<'a> for &'a UniformBuffer<T> {
+    #[inline]
+    fn into_binding(self) -> BindingResource<'a> {
+        BindingResource::Buffer(
+            self.buffer()
+                .expect("Failed to get buffer")
+                .as_entire_buffer_binding(),
+        )
```

Comment:
This could also work. Up to you which one you prefer

```suggestion
        self.buffer()
            .expect("Failed to get buffer")
            .as_entire_buffer_binding()
            .into_binding(),
```

Reply:
Oh, sorry, the comma should have been remove 😅